impeller/renderer/backend/vulkan/test/mock_vulkan.cc

```diff
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 #include "impeller/renderer/backend/vulkan/test/mock_vulkan.h"
+#include "vulkan/vulkan_core.h"
```

Comment:
Import the `//impeller/renderer/backend/vulkan.vk.h` umbrella header.

Reply:
Done.